package.json

```diff
@@ -104,13 +103,14 @@
     "react-redux": "7.1.0",
     "react-resize-detector": "3.4.0",
     "react-router-dom": "5.0.1",
-    "react-scripts": "3.1.1",
+    "react-scripts": "3.2.0",
     "redux": "4.0.1",
     "redux-persist": "5.10.0",
     "redux-persist-transform-filter": "0.0.18",
     "redux-thunk": "2.3.0",
     "reselect": "4.0.0",
     "ssri": "6.0.1",
+    "table-resolver": "3.3.0",
```

Comment:
Can you explain what `table-resolver` is?

Reply:
Same.

Reply:
I don't exactly know what is it :)
What I know, is that is being imported here: https://github.com/kiali/kiali-ui/blob/master/src/pages/Graph/GraphHelpFind.tsx#L5

...and used to build a table in that source file.

Reply:
As I've already said, the GraphHelpFind control uses table-resolver. And I see that it hasn't been migrated to PF4 yet. 
I see that @jshaughn is working on the migration on #1861. We could wait for him, and I'll rebase and omit this dep, or merge this PR and ask him to do the removal of table-resolver dependency.

Reply:
Mine should be in today...